src/system.rs

```diff
@@ -290,6 +290,28 @@ pub fn list_cpu_governors() -> Vec<String> {
     list_cpus().into_iter().map(|x| x.gov).collect()
 }
 
+pub fn read_int(path: &str) -> Result<i32, Error> {
```

Comment:
Good call moving these

Reply:
Yeah I noticed that they would be duplicate with the thermal zone reader. We also need to refactor the cpus code it's a little wacky I feel like. That's an issue for another time.